yelp_beans/routes/tasks.py

```diff
@@ -92,3 +96,31 @@ def send_match_emails():
         logging.info(matches)
         send_batch_meeting_confirmation_email(matches, spec)
     return "OK"
+
+
+@tasks.route('/init', methods=['GET'])
+def init():
```

Comment:
Explored the option of using a script with the [remote api](https://cloud.google.com/appengine/docs/standard/python/tools/remoteapi).  I was attempting to cut down on the user complexity of having to add `remote_api: on` to the app.yaml, deploying, running a script, removing the line, and deploying again.

The route is only accessible by admins and is a noop if there is data, while not great from an engineering perspective, it seems to be the quickest upstart from a user perspective.